builder/exector/export_app.go

```diff
@@ -571,7 +571,7 @@ func (i *ExportApp) buildDockerComposeYaml() error {
 			volumeName = buildToLinuxFileName(volumeName)
 			volumePath := item.Get("volume_path").String()
 			if item.Get("volume_type").String() == "config-file" {
-				volume := fmt.Sprintf("__GROUP_DIR__/%s/%s:%s", appName, volumePath, volumePath)
+				volume := fmt.Sprintf(".%s:%s", path.Join("./", appName, volumePath), volumePath)
```

Comment:
The appName maybe Chinese, so it is not appropriate to appear directly in the path.

Reply:
appName will be converted to pinyin